functions/index.js

```diff
@@ -1,5 +1,9 @@
 import admin from 'firebase-admin';
-import { onRequest } from 'firebase-functions/v2/https';
+import { onCall, onRequest } from 'firebase-functions/v2/https';
```

Comment:
**todo**: i assume you'll need a security exception for this?

Reply:
I already applied for one for the old functions I moved away from. But maybe they added a project-wide exception because I was able to publish these new functions without any issues. ¯\_(ツ)_/¯

I'm sure I'll have to apply for the security exception for the prod project. But it should go fairly smoothly since I've got the Archer package updated.